src/multiselect-combo-box.js

```diff
@@ -37,7 +37,7 @@ const multiselectComboBox = css`
 `;
 
 registerStyles('multiselect-combo-box', [inputFieldShared, multiselectComboBox], {
-  moduleId: 'multiselect-combo-box-styles'
+  moduleId: 'vaadin-multiselect-combo-box-styles'
```

Comment:
Should this prefix be applied to the other registered styles as well? As potentially we could encounter similar issues?

cc @web-padawan What do you think? Would adding a `vaadin` prefix cause any issues when used by an add-on component?

Reply:
Good catch. Yes, we need this prefix for styles in `src` to apply before themes.

Reply:
@gatanaso Yes, I'm done! PTAL.